lib/session.js

```diff
@@ -32,7 +30,6 @@ module.exports = class Session {
 
   regenerate (request) {
     this.sessionId = this[generateId](request)
-    this.encryptedSessionId = this[sign]()
```

Comment:
instead of removing it from `Session`, what about just ensuring it's not saved to the store? I.e. defining a custom `toJSON` or something